### PR TITLE
docs - awscc_apigateway_method

### PR DESCRIPTION
**Pull request by dansantf**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: templates/resources/apigateway_method.md.tmpl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### First example
```
Comment (wellsiau-aws) [inline templates/resources/apigateway_method.md.tmpl:14]: can you please update this sub-heading with title of your example

File: templates/resources/apigateway_method.md.tmpl [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Description about the first example
+{{ tffile (printf "examples/resources/%s/apigateway_method_basic_example.tf" .Name)}}
+
+### Second example
```
Comment (wellsiau-aws) [inline templates/resources/apigateway_method.md.tmpl:18]: same here

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
thanks for your contribution, please take a look at the suggested changes
once you made the changes, please re-run `make docs`

File: templates/resources/apigateway_method.md.tmpl (hunk 2 above)
Comment (dansantf) [reply templates/resources/apigateway_method.md.tmpl]: Hey! Thank you for the review, just added the changes you mentioned

File: docs/resources/apigateway_method.md [hunk 3]
```diff
@@ -10,7 +9,61 @@ description: |-
 
 Resource Type definition for AWS::ApiGateway::Method
 
+## Example Usage
+
+### Basic use of API method response
+Description about the first example
```
Comment (wellsiau-aws) [inline docs/resources/apigateway_method.md:15]: you can remove this line or use it to expand the description about the example

File: docs/resources/apigateway_method.md [hunk 4]
```diff
@@ -10,7 +9,61 @@ description: |-
...
+  resource_id = awscc_apigateway_rest_api.terraform_apigateway_rest_api.root_resource_id
+
+  method_responses = [{ status_code = "200" }]
+
+  depends_on = [awscc_apigateway_rest_api.terraform_apigateway_rest_api]
+}
+```
 
+### Use of API method response with response parameters and response models
+Description about the second example
```
Comment (wellsiau-aws) [inline docs/resources/apigateway_method.md:42]: you can remove this line or use it to expand the description about the example

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
thanks for making the previous changes, just another small nit

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thank you @dansantf

**Comment by wellsiau-aws** [comment]
@dansantf , sorry for the delay. Looks like there's previous PR that also added some examples for API Gateway Method, thus your PR now contain conflicting files.

May I suggest you to re-base your branch and resolve the conflict and push new update here? thank you.

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
I noticed other resources was included on the latest push including `awscc_personalize_dataset` and `awscc_personalize_dataset_group`. Please take this off from the PR. Thank you

**Comment by wellsiau-aws** [comment]
closing this in favor of the new PR #1559

**Pull request closed without merging**